src/viam/ur/module/ur_arm.cpp

```diff
@@ -508,48 +1549,6 @@ void URArm::check_configured_(const lock_type<std::shared_mutex>&) {
     }
 }
 
-void URArm::trajectory_done_cb_(const control::TrajectoryResult state) {
```

Comment:
This mostly just moved, and is now part of `struct state_`, so it moved in with the other methods of that class.